test_runner/fixtures/safekeeper/http.py

```diff
@@ -170,7 +193,10 @@ def timeline_status(
         res.raise_for_status()
         resj = res.json()
         walreceivers = [Walreceiver(wr["conn_id"], wr["status"]) for wr in resj["walreceivers"]]
+        # It is always normally not None, it is allowed only to make forward compat tests happy.
+        mconf = Configuration.from_json(resj["mconf"]) if "mconf" in resj else None
```

Comment:
This still causes `KeyError: 'mconf'` to be issued if not present, causing some test failures (see test bot comment above).

Reply:
Weird, probably bot failed to update comment? All checks are green currently, and I don't see how it can fail this way.